### PR TITLE
PR from rebase

### PR DESCRIPTION
**Pull request by Gopi1214**
catalogue issue resolved
logins created successfully

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -10,4 +10,5 @@ This is my first commit to my new repo.
 
 ### rebase
 * rebase is another merge strategy.
-* still developping the code.
\ No newline at end of file
+* still developping the code.
```
Comment (Mahesh8120) [inline README.MD:13]: is this required

Comment (Gopi1214) [reply README.MD:13]: yes it is

Comment (Mahesh8120) [reply README.MD:13]: okay

**Review by Mahesh8120 — APPROVED** [review]
everything is okay I am approving.

**Comment by Gopi1214** [comment]
merge

**Pull request merged**